packages/js/product-editor/src/components/number-control/number-control.tsx

```diff
@@ -71,10 +86,59 @@ export const NumberControl: React.FC< NumberProps > = ( {
 				{ ...inputProps }
 				step={ step }
 				disabled={ disabled }
+				autoComplete="off"
 				id={ id }
-				suffix={ suffix }
+				className="woocommerce-number-control"
+				suffix={
+					<>
+						{ suffix }
+						{ isFocused && (
+							<>
+								<Button
+									className="woocommerce-number-control__increment"
+									icon={ plus }
+									onClick={ () =>
+										onChange(
+											String(
+												parseFloat( value || '0' ) +
+													step
+											)
+										)
+									}
+									onBlur={ unfocusIfOutside }
+									isSmall
+									aria-hidden="true"
+									aria-label={ __(
+										'Increment',
+										'woocommerce'
+									) }
+									tabIndex={ -1 }
+								/>
+								<Button
+									icon={ reset }
```

Comment:
This is a little fragile, as there is no inherent link between the symbolism of "reset" and the minus operator.

It would probably be safer to just use the `+` and `-` characters for these buttons.

Reply:
I understand your point, but I think that's ok to use this icon. It's the same one used by Gutenberg [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/number-control/index.tsx#L257).